cognite/client/data_classes/iam.py

```diff
@@ -119,33 +119,44 @@ def dump(self, camel_case: bool = False) -> dict[str, Any]:
         return dumped
 
 
-class CreatedSession(CogniteResource):
+class CreatedSession(CogniteResponse):
     """Session creation related information
 
     Args:
-        id (int | None): ID of the created session.
+        id (int): ID of the created session.
+        status (str): Current status of the session.
+        nonce (str): Nonce to be passed to the internal service that will bind the session
         type (str | None): Credentials kind used to create the session.
-        status (str | None): Current status of the session.
-        nonce (str | None): Nonce to be passed to the internal service that will bind the session
         client_id (str | None): Client ID in identity provider. Returned only if the session was created using client credentials
-        cognite_client (CogniteClient | None): No description.
     """
 
     def __init__(
         self,
-        id: int | None = None,
+        id: int,
+        status: str,
+        nonce: str,
```

Comment:
Technically a breaking change?

Reply:
Yes, but I honestly think it's fine as there is no reason for a user to instantiate this themselves... 😬

Reply:
agreed.